diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/rules/NoSpacesRule.kt

```diff
@@ -0,0 +1,89 @@
+package org.cqfn.diktat.ruleset.rules
+
+import com.pinterest.ktlint.core.KtLint
+import com.pinterest.ktlint.core.Rule
+import com.pinterest.ktlint.core.ast.ElementType.ENUM_ENTRY
+import com.pinterest.ktlint.core.ast.ElementType.WHITE_SPACE
+import com.pinterest.ktlint.core.ast.isWhiteSpaceWithNewline
+import com.pinterest.ktlint.core.ast.parent
+import org.cqfn.diktat.common.config.rules.RuleConfiguration
+import org.cqfn.diktat.common.config.rules.RulesConfig
+import org.cqfn.diktat.common.config.rules.getRuleConfig
+import org.jetbrains.kotlin.com.intellij.lang.ASTNode
+import org.cqfn.diktat.ruleset.constants.Warnings.TOO_MANY_SPACES
+import org.jetbrains.kotlin.com.intellij.psi.impl.source.tree.LeafElement
+
+class NoSpacesRule : Rule("no-spaces") {
```

Comment:
Why `NoSpacesRule`, when it actually is `TooManySpacesRule`? Or even, `ConsecutiveSpacesRule`.